packages/react-data-grid/src/masks/__tests__/InteractionMasks.spec.js

```diff
@@ -222,37 +224,37 @@ describe('<InteractionMasks/>', () => {
         it('should shrink the selection upwards on Shift+Up', () => {
           pressKey(wrapper, 'ArrowUp', { shiftKey: true });
           let selectedRange = wrapper.state('selectedRange');
-          expect(selectedRange.topLeft).toEqual({ idx: 2, rowIdx: 2 });
-          expect(selectedRange.bottomRight).toEqual({ idx: 3, rowIdx: 2 });
-          expect(selectedRange.cursorCell).toEqual({ idx: 3, rowIdx: 2 });
-          expect(selectedRange.startCell).toEqual({ idx: 2, rowIdx: 2 });
+          expect(selectedRange.topLeft).toEqual(objectMatching({ idx: 2, rowIdx: 2 }));
```

Comment:
Is the object being tested changed? Just figuring out the need for `objectMatching`

Reply:
It now has a `changeRowOrColumn` property that it is expecting. Not sure why the tests didnt fail before. Since I'm only interested in the cell position I used `objectContaining`